yarn.lock

```diff
@@ -0,0 +1,2915 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```

Comment:
Please use `npm`'s `package-lock.json`, rather than `yarn.lock` (for consistency with the rest of this repository).